tests/auth_utils.py

```diff
@@ -87,7 +87,7 @@ class EnvironmentVariables:
     LOGIN_FLOW: _LOGIN_FLOW = "infer"
     IDP_CLIENT_ID: str | None = None
     IDP_CLIENT_SECRET: str | None = None
-    TOKEN: str | None = None
+    CDF_TOKEN: str | None = None
```

Comment:
Small bug in the `get_toolkit_client` which is not part of the Toolkit package, but used in manual testing.